src/entities/mellow/mellowLpRouter.ts

```diff
@@ -24,6 +25,34 @@ import { abi as IERC20MinimalABI } from '../../ABIs/IERC20Minimal.json';
 import { abi as MellowMultiVaultRouterABI } from '../../ABIs/MellowMultiVaultRouterABI.json';
 import { sentryTracker } from '../../utils/sentry';
 import { closeOrPastMaturity, MellowProductMetadata } from './config';
+import { convertGasUnitsToUSD } from '../../utils/mellowHelpers/convertGasUnitsToUSD';
+
+const geckoEthToUsd = async () => {
+  for (let attempt = 0; attempt < 5; attempt += attempt) {
+    try {
+      const data = await axios.get(
+        `https://pro-api.coingecko.com/api/v3/simple/price?x_cg_pro_api_key=${process.env.REACT_APP_COINGECKO_API_KEY}&ids=ethereum&vs_currencies=usd`,
+      );
+      return data.data.ethereum.usd;
+    } catch (error) {
+      console.log(error);
+    }
+  }
+  return 0;
+};
+
+const getGasPriceGwei = async () => {
+  for (let attempt = 0; attempt < 5; attempt += attempt) {
```

Comment:
attempt += attempt? 

Also worth moving this function to utils.

Reply:
fixed